app/src/components/EditEvent/EditEventContainer.tsx

```diff
@@ -49,30 +52,21 @@ export const EditEventContainer = () => {
     }
   }, [eventId, editToken, saveEditableEvents]);
 
-  if (!event || !eventId) {
+  if (!hasLoaded(remoteEvent)) {
     return <div>Loading</div>;
   }
 
+  if (!event || !eventId) {
+    return <div>Kan ikke finne event med id {eventId}</div>;
+  }
+
   const putEditedEvent =
-    isOk(event) &&
+    validEvent &&
     catchAndNotify(async () => {
-      if (event.validValue.participantQuestion === '') {
-        updateEvent({
-          ...event.editValue,
-          participantQuestion: parseQuestion(
-            'Allergier, preferanser eller noe annet på hjertet?'
-          ),
-        });
-      }
```

Comment:
Vi kan godt sette default spørsmål et annet sted, men vi må sette det en plass - kan ikke ha tomt spm for da gir ikke svarene noe mening.

Reply:
Ja her gikk det nok litt fort i svingene, var ikkje meininga å berre fjerne den. Men ser ikkje korleis dette har fungert? Slik `useState` funker på kan ikkje denne oppdateringa ha blitt med i nettverkskallet under^?